src/encoded/static/components/app.js

```diff
@@ -277,6 +278,7 @@ const fullPageReloadPaths = [
     '/summary/',
     '/encyclopedia/',
     '/immune-cells/',
+    /human-donor-matrix/,
```

Comment:
I was hoping to never add to this array, and instead fix whatever bug causes a required reload. But as I don’t understand why these pages require a full-page reload, I guess we have to do this. Not sure how this works without the single quotes though? Does it interpret this as a regular expression or something?

Reply:
Removed as it is not needed. I was fixing an issue and throwing mud to try to figure it out. I should of had removed this after I figured it out. If curious, the issue was I had three radio buttons with two of them both having checked={true} based on a certain click-pattern and it was causing one button not to work.